docs/index.mdx

```diff
@@ -1,42 +0,0 @@
-# Coffee Plugin Manager
-
-Welcome to **Coffee**, the plugin manager for Core Lightning. 
-It takes care of all the configuration and installation of a plugin for your Core Lightning node. 
-
-Using Coffee, anyone can manage plugins for their Core Lightning node. 
-Coffee handles all the tedious setup and configuration tasks of the underlying 
-Core Lightning plugin infrastructure and the hassle of setting up dependencies.
```

Comment:
this needs to stay, it is used to build the https://docs.page/coffee-tools/coffee

Reply:
Yes. I didn't notice